src/hidpp20.c

```diff
@@ -2959,6 +2958,10 @@ hidpp20_onboard_profiles_write_profile(struct hidpp20_device *device,
 
 	memcpy(pdata->profile.name.txt, profile->name, sizeof(profile->name));
 
+	// This field needs to be zero for leds to work in G705
+	// If a non zero value is set, custom animation will be used instead
```

Comment:
I think this comment should go above the `custom_animation_g705` struct field instead.